native/src/main/java/io/ballerina/stdlib/oracledb/Constants.java

```diff
@@ -176,6 +178,24 @@ private Types() {
 
         }
 
+        public static final String INTERVAL_YEAR_TO_MONTH_RECORD = "IntervalYearToMonth";
+        public static final String INTERVAL_DAY_TO_SECOND_RECORD = "IntervalDayToSecond";
+
+        /**
+         * Constants related to Oracle Database specific sql type int values.
+         */
+
+        public static final class OracleDbSpecificSqlTypes {
+
+            private OracleDbSpecificSqlTypes() {}
+
+            public static final int TIMESTAMP_WITH_TIME_ZONE = -101;
+            public static final int TIMESTAMP_WITH_LOCAL_TIME_ZONE = -102;
+            public static final int INTERVAL_YEAR_TO_MONTH = -103;
+            public static final int INTERVAL_DAY_TO_SECOND = -104;
```

Comment:
Are these constants given in the driver. 
IMO better to import the constants from the driver else, we have to check these periodically after each driver release

Reply:
Could not find driver constants in the sql data types documentation. https://docs.oracle.com/en/database/oracle/oracle-database/12.2/sqlqr/Data-Types.html#GUID-DCCC6F18-15A0-4ECC-BA48-16F73F844844.

Reply:
Not in the guide but in their code. There should be a similar class to `SQLTypes` in JDBC package. Lets verify

Reply:
Driver types found and added in https://github.com/ballerina-platform/module-ballerinax-oracledb/pull/13/commits/2ea384e5a8316fddb2335f302fc800b6931906a5